k8s/testground-daemon/deployment.yml

```diff
@@ -0,0 +1,71 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: testground-daemon
+  labels:
+    app: testground-daemon
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: testground-daemon
+  template:
+    metadata:
+      labels:
+        app: testground-daemon
+    spec:
+      serviceAccountName: testground-daemon
+      hostPID: true
+      hostNetwork: true
```

Comment:
@coryschwartz this is rather ugly, but I couldn't come up with a better solution for setting up `goproxy`.

We need the `goproxy` to be accessible from a `build container` started by `testground daemon`. Since `build containers` are not started by Kubernetes, they don't have access to DNS, so the only way I could configure the setup to has a `goproxy` is if I know the goproxy is on the same host on a specific port (in this case 8081).

---

I am really against the daemon starting containers on the host that are long-running such as the goproxy, so this is one way of how we could deploy a goproxy (currently with mount to EFS), so that we don't download the same deps multiple times.

Reply:
I suppose this is why the goproxy service file is there -- were you trying to set this up as two different pods which communicate through the service? What prevented that from working?